### PR TITLE
doc for kubectl-gs cluster updates

### PR DESCRIPTION
**Pull request by njuettner**
This PR introduces documentation for executing cluster updates via `kubectl-gs`

Adds documentation for PR https://github.com/giantswarm/kubectl-gs/pull/613

### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [ ] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/main/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

File: src/content/reference/kubectl-gs/update-clusters.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+### Schedule a cluster update 
+
+```nohighlight
+kgs update cluster --name a1b2c --namespace org --release-version 16.1.0 --scheduled-time "2022-01-01 02:00" --provider aws
+```
+
+Supported flags:
+
+- `--name` - ID of the cluster.
```
Comment (njuettner) [inline src/content/reference/kubectl-gs/update-clusters.md:32]: Shall I change it to `cluster-id`. We currently have different flags for the same thing, looking at the docs for `kubectl-gs` we use `cluster-id` and `cluster`. 
I'm favour of using `cluster-id` here, WDYT?

**Review by MarcelMue — APPROVED** [review]

File: src/content/ui-api/kubectl-gs/update-cluster.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+linkTitle: update cluster 
+title: "'kubectl gs update cluster' command reference"
+description: Reference documentation on how to update update a Cluster using 'kubectl gs'.
```
Comment (njuettner) [inline src/content/ui-api/kubectl-gs/update-cluster.md:4]:
```suggestion
description: Reference documentation on how to update a Cluster using 'kubectl gs'.
```

File: src/content/ui-api/kubectl-gs/update-cluster.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+---
+linkTitle: update cluster 
+title: "'kubectl gs update cluster' command reference"
+description: Reference documentation on how to update a Cluster using 'kubectl gs'.
+weight: 100
+menu:
+  main:
+    parent: uiapi-kubectlgs
```
Comment (marians) [inline src/content/ui-api/kubectl-gs/update-cluster.md:8]:
```suggestion
    identifier: kubectl-gs-update-cluster
    parent: uiapi-kubectlgs
```

This is required in this case to make the item appear in the menu. Without this, HUGO issues a warning

```
WARN 2021/12/14 11:11:09 "/Users/marian/git/giantswarm/docs/src/content/ui-api/kubectl-gs/update-cluster.md:1:1": duplicate menu entry with identifier "update cluster" in menu "main"
```

and does not add the page to the menu.

File: src/content/ui-api/kubectl-gs/update-cluster.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+last_review_date: 2021-12-13
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-phoenix
+user_questions:
+  - How can I update a cluster from the command line?
+---
+
+# `kubectl gs update cluster`
+
+Updating clusters can be done in two ways. A cluster can be immediately updated or a cluster update can be scheduled in the future.
```
Comment (marians) [inline src/content/ui-api/kubectl-gs/update-cluster.md:20]:
```suggestion
This command allows to upgrade a workload cluster. Upgrades can either be triggered immediately, or the upgrade can be scheduled to happen at a specific date and time.
```

File: src/content/ui-api/kubectl-gs/update-cluster.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+## Usage
+
+### Update a cluster immediately
```
Comment (marians) [inline src/content/ui-api/kubectl-gs/update-cluster.md:24]:
```suggestion
### Upgrade a cluster immediately
```

File: src/content/ui-api/kubectl-gs/update-cluster.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+```nohighlight
+kgs update cluster --name a1b2c --namespace org --release-version 16.1.0 --provider aws
+```
+
+### Schedule a cluster update
```
Comment (marians) [inline src/content/ui-api/kubectl-gs/update-cluster.md:30]:
```suggestion
### Schedule a cluster upgrade
```

File: src/content/ui-api/kubectl-gs/update-cluster.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+# `kubectl gs update cluster`
+
+Updating clusters can be done in two ways. A cluster can be immediately updated or a cluster update can be scheduled in the future.
+
+## Usage
+
+### Update a cluster immediately
+
+```nohighlight
+kgs update cluster --name a1b2c --namespace org --release-version 16.1.0 --provider aws
```
Comment (marians) [inline src/content/ui-api/kubectl-gs/update-cluster.md:27]: - We don't use the `kgs` alias any more.
- Long commands should be broken into one flag per line
- Better use a more realistic example namespace

```suggestion
kubectl gs update cluster \
  --name a1b2c \
  --namespace org-acme \
  --release-version 16.1.0 \
  --provider aws
```

File: src/content/ui-api/kubectl-gs/update-cluster.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+### Schedule a cluster update
+
+```nohighlight
+kgs update cluster --name a1b2c --namespace org --release-version 16.1.0 --scheduled-time "2022-01-01 02:00" --provider aws
+```
+
+Supported flags:
+
+- `--name` - ID of the cluster.
```
Comment (marians) [inline src/content/ui-api/kubectl-gs/update-cluster.md:38]:
```suggestion
- `--name` - Name of the cluster.
```

File: src/content/ui-api/kubectl-gs/update-cluster.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+- `--namespace` - Namespace of the cluster.
```
Comment (marians) [inline src/content/ui-api/kubectl-gs/update-cluster.md:39]:
```suggestion
- `--namespace` - Namespace of the cluster resource.
```

File: src/content/ui-api/kubectl-gs/update-cluster.md [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+- `--release-version` - Version of the release the cluster should be upgraded to.
+- `--scheduled-time` - Optionally: Scheduled time when cluster should be updated, time format 'YYYY-MM-DD HH:MM'.
```
Comment (marians) [inline src/content/ui-api/kubectl-gs/update-cluster.md:41]:
```suggestion
- `--scheduled-time` (optional): Scheduled time when cluster should be updated, in the format 'YYYY-MM-DD HH:MM'. Timezone UTC is assumed. If not given, the upgrade happens as soon as possible.
```

File: src/content/ui-api/kubectl-gs/update-cluster.md [hunk 11]
```diff
@@ -0,0 +1,47 @@
+---
+linkTitle: update cluster 
+title: "'kubectl gs update cluster' command reference"
+description: Reference documentation on how to update a Cluster using 'kubectl gs'.
```
Comment (marians) [inline src/content/ui-api/kubectl-gs/update-cluster.md:4]:
```suggestion
description: Reference documentation on how to upgrade a workload cluster using kubectl-gs.
```

**Review by marians — COMMENTED** [review]
Lots of little detail comments. Anyway, good work!

**Review by marians — APPROVED** [review]

**Pull request merged**